src/base/bittorrent/tracker.cpp

```diff
@@ -34,6 +34,7 @@
 #include <libtorrent/entry.hpp>
 
 #include <QHostAddress>
+#include <qnamespace.h>
```

Comment:
Why is it for?
Anyway you shouldn't include this file directly. There is `Qt` header: https://doc.qt.io/qt-5/qt.html

Reply:
@glassez It got included due to me using `iwyu` mode in my environment. I changed it to `#include <Qt>`, but let me know if this is bad practice and should be deleted.

Reply:
>let me know if this is bad practice and should be deleted.

I think including this header here is redundant since it's included indirectly and you don't use its members independently.

Reply:
@glassez 

The point is precisely to not include things indirectly. `Qt::CaseInsensitive` is defined in the `Qt` header file, so it is directly included.

Reply:
>The point is precisely to not include things indirectly. 

Why not?
`QString` uses `Qt::CaseSensitivity` in its interface and you use it in context of using `QString` only...

Reply:
@glassez As you wish, removed.